### ISSUE TITLE
Umbraco v15 tried to install package Diplo.AuditLogViewer

### ISSUE THREAD

**Issue by fmulyono** [umbraco/Umbraco-CMS#17840]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

15.0.0

### Bug summary

I installed Umbraco v15.0.0 in my local machine and started development for a few days. Everything works fine (No package installed)
Just now, I tried to install package Diplo.AuditLogViewer. After installing the package and restarted the development server, I got this error:

failed with 3 error(s) (9.8s)
    Program.cs(4,6): error CS0121: The call is ambiguous between the following methods or properties: 'Umbraco.Extensions.UmbracoBuilderExtensions.AddBackOffice(Umbraco.Cms.Core.DependencyInjection.IUmbracoBuilder, System.Action<Microsoft.Extensions.DependencyInjection.IMvcBuilder>?)' and 'Umbraco.Extensions.UmbracoBuilderExtensions.AddBackOffice(Umbraco.Cms.Core.DependencyInjection.IUmbracoBuilder, System.Action<Microsoft.Extensions.DependencyInjection.IMvcBuilder>?)'
    Program.cs(17,11): error CS0121: The call is ambiguous between the following methods or properties: 'Umbraco.Extensions.UmbracoApplicationBuilderExtensions.UseBackOffice(Umbraco.Cms.Web.Common.ApplicationBuilder.IUmbracoApplicationBuilderContext)' and 'Umbraco.Extensions.UmbracoApplicationBuilderExtensions.UseBackOffice(Umbraco.Cms.Web.Common.ApplicationBuilder.IUmbracoApplicationBuilderContext)'
    Program.cs(22,11): error CS0121: The call is ambiguous between the following methods or properties: 'Umbraco.Extensions.UmbracoApplicationBuilderExtensions.UseBackOfficeEndpoints(Umbraco.Cms.Web.Common.ApplicationBuilder.IUmbracoEndpointBuilderContext)' and 'Umbraco.Extensions.UmbracoApplicationBuilderExtensions.UseBackOfficeEndpoints(Umbraco.Cms.Web.Common.ApplicationBuilder.IUmbracoEndpointBuilderContext)'

Tried to upgrade to Umbraco v15.1.1, but the error remains

### Steps to reproduce

1. Install Umbraco v15.0.0
2. Execute this command: dotnet add package Diplo.AuditLogViewer
3. Restart the server

**Comment by PerplexDaniel** [comment]
The newest version of that package [has a dependency](https://www.nuget.org/packages/Diplo.AuditLogViewer#dependencies-body-tab) on Umbraco v10. It probably does not work at all in v15.

![image](https://github.com/user-attachments/assets/9614eebb-52f5-47af-b4cf-9ef14cb27202)

**Comment by nul800sebastiaan** [comment]
Indeed, looks like it's not been updated for v14+ yet - https://marketplace.umbraco.com/package/diplo.auditlogviewer
